external/malmo/Minecraft/run/recipe_dump.txt

```diff
@@ -0,0 +1,441 @@
+1xtile.blockLapis = 9xitem.dyePowder.blue(Lapis Lazuli)
```

Comment:
Please see the comment in https://github.com/ml4ai/tomcat/pull/21/files#r342582783.